bigcache.go

```diff
@@ -46,20 +47,26 @@ func newBigCache(config Config, clock clock) (*BigCache, error) {
 		config.Hasher = newDefaultHasher()
 	}
 
+	maxShardSize := 0
+	if config.HardMaxCacheSize > 0 {
+		maxShardSize = config.HardMaxCacheSize * 1024 * 1024 / config.Shards
```

Comment:
Magic numbers? It's not obvious why you are multiplying by 1024. Maybe add unit to `maxHardSize`